fs/rails-base
# Add feedbacks

# Summary

As Visitor I want to leave feedback
As User I want to see pre-filled name and email in feedback form

# Test plan

* Go to /feedbacks/new
* Fill in the following fields: name, email, text
* Click on "Submit feedback" button
* You should see that "Feedback was successfully send!"

If user signed in
* You should see pre-filled with email and name on feedback page (/feedbacks/new)

# Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduces DB fields have indexes and constraints
- routes are RESTful, no useless routes
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

# Deploy notes

None
